Mathlib/Algebra/Star/Subalgebra.lean

```diff
@@ -340,17 +339,14 @@ variable [Semiring A] [Algebra R A] [StarRing A] [StarModule R A]
 variable [Semiring B] [Algebra R B] [StarRing B] [StarModule R B]
 
 /-- The pointwise `star` of a subalgebra is a subalgebra. -/
-instance involutiveStar : InvolutiveStar (Subalgebra R A)
-    where
+instance involutiveStar : InvolutiveStar (Subalgebra R A) where
   star S :=
     { carrier := star S.carrier
-      mul_mem' := @fun x y hx hy =>
-        by
+      mul_mem' := fun {x y} hx hy => by
```

Comment:
Is there a way to use `where` syntax for this kind of thing too?

Reply:
If you find a way, let me know :)